Adyen/HttpClient/HttpWebRequestWrapper.cs

```diff
@@ -223,5 +206,9 @@ public static string QueryString(IDictionary<string, string> dict)
             }
             return string.Join("&", list);
         }
+
+        public void Dispose()
```

Comment:
Is this needed?

Reply:
It's needed for backward compatibility only since the new implementation uses `IDisposable` pattern and the users of `IClient` interface shouldn't take care of that (just implemented encapsulation principle).

Reply:
The main idea was about giving backward compatibility for library users. So if they don't use HttpClient yet everything should work more or less the same way. With this approach, the version upgrade should be softer, without breaking changes. :)